src/monitor/node_metadata.h

```diff
@@ -90,6 +90,7 @@ typedef enum SyncState
  * candidate.
  */
 #define MAX_USER_DEFINED_CANDIDATE_PRIORITY 100
+#define CANDIDATE_PRIORITY_INCREMENT (MAX_USER_DEFINED_CANDIDATE_PRIORITY + 1)
```

Comment:
What was this `+ 1` needed for?

Reply:
It used to be that we would only allow failover when candidate priority was in the range 1..100, and now we allow it when it's in the range 0..100. We want to get from 0 to 101, and then from 101 back to 0...